docs/r3/beyondcandc.md

```diff
@@ -1,28 +1,29 @@
 # Beyond "command and control"
+
 "Command and control"-style interoperation, where system A instructs system B to do something, or makes decisions based on observable properties of system B, is largely a solved problem at the high level. There is always work to do in defining message standards and communications protocols, but the basic co-ordination mechanisms are well-understood:
 
 * **RPC (synchronous)**: A sends a request to B, which B executes immediately, returning a response indicating the outcome of the request.
 * **RPC (asynchronous)**: A sends a request to B, which B reliably queues for processing. A may poll for updates on the processing status of the request, or B may send a message to A when an outcome is reached.
 * **Event stream subscription**: A sends a subscription request to B, asking to be notified of events of a certain type, and receives a subscription reference in response. A may then either poll the subscription for new events, or receive a stream of messages from B providing updates as they occur.
 
-The establishment of suitable command-and-control methods for systems to query, instruct and observe one another is necessary but not sufficient for interoperation when the requirements include reliable synchronisation of state changes across networks without a trusted central co-ordinator. Building on these primitives, we also require:
+The establishment of suitable command-and-control methods for systems to query, instruct and observe one another is _necessary but not sufficient_ for interoperation when requirements include reliable synchronisation of state changes across networks without a trusted central co-ordinator. Building on these primitives, we also require:
```

Comment:
Should this be the firs thing we mention in the section? I wonder if we're not burying the lead a little. I'm not sure at the least if we're adequatly detailing why we feel this is inherantly sub optimal solution